beacon_node/network/src/sync/range_sync/chain.rs

```diff
@@ -1117,6 +1167,15 @@ impl<T: BeaconChainTypes> SyncingChain<T> {
             return None;
         }
 
+        // don't send batch requests until we have peers on custody subnets
+        if !self.good_peers_on_custody_subnets(self.to_be_downloaded, network) {
+            debug!(
+                self.log,
+                "Waiting for peers to be available on custody column subnets"
+            );
+            return None;
+        }
```

Comment:
This workaround can be removed once individual requests in range sync are made retryable in https://github.com/sigp/lighthouse/issues/6258, perhaps worth adding a comment here

Reply:
Added comment